### DISCUSSION TITLE
Now Available ✨ Close issue as a duplicate, REST API for sub-issues, and more!

### DISCUSSION THREAD

**Discussion by github-staff** [community/community#146942] [Projects and Issues]
Following our "Evolving GitHub Issues" [announcement](https://github.blog/changelog/2024-10-01-evolving-github-issues-public-beta/) we've continued to improve the experience based on your feedback, including closing an issue as a duplicate, a REST API for sub-issues, and expanding the limits for both sub-issues and issue types.

These new features are all available in public preview for you to try. To gain access for your organization, please sign up [here](https://github.com/features/issues/signup). Check out the changelog announcement [here](https://github.blog/changelog/2024-12-12-github-issues-projects-close-issue-as-a-duplicate-rest-api-for-sub-issues-and-more/). 

## :broom: Close an issue as a duplicate

You can now close an issue as a duplicate of another issue, making it easier to manage your issues and provide more clarity on why they were closed.

When closing an issue, select `Close as duplicate` from the dropdown to search for and select the duplicate issue. You'll then see an event in the timeline and note at the top making it clear why it was closed.

https://github.com/user-attachments/assets/a4bbbb3c-afda-4dde-b356-210ea8e2ab6c

## :gear: REST API support for sub-issues

You can now use the REST API to view, add, remove, and reprioritize sub-issues, making it easier to automate your use of sub-issues. Check out the [documentation](https://docs.github.com/en/rest/issues/sub-issues?apiVersion=2022-11-28) to learn more.

## :heavy_plus_sign: Increased limits for sub-issues and issue types

You can now have up to 100 sub-issues per parent issue (up from 50), as well as up to 25 issue types in an organization (up from 10), making it easier to manage, classify, and break down work.

![Issue type organization settings showing maximum limit of 25 issue types](https://github.com/user-attachments/assets/3ddef1c7-2f8c-48f0-87af-22a38b4610f3)

## :iphone: Issue types on GitHub Mobile 

You can now view, add, and update issue types on GitHub Mobile.

![Issue types on GitHub Mobile](https://github.com/user-attachments/assets/40bc9b4a-a6cc-434c-9a2d-a4b314f99e3f)

## :mag: Improved filtering for sub-issues and issue types

You can use the `has:` and `no:` filters to search for sub-issues and issue types both from a project and the repository issues page, making it easier to find the exact set of issues you're looking for and make updates.

![Issue filtering using has filter](https://github.com/user-attachments/assets/27b83b7d-3d46-4d6a-8418-afbfd3356def)

Example filters include:
- `no:type` to find all issues that do not yet have a type
- `no:parent-issue` to find all issues without a parent issue
- `has:sub-issue` to find all issues that have sub-issues

## :sparkles: Additional improvements
On top of the many bug fixes we've shipped, we've also introduced the following improvements:
- If the sub-issue is from a different repository than the parent issue, you will now see the repository name in the sub-issues list. 
- In GitHub markdown, pasting in a project link will now show the project name as well as more project details on hover. 
- Projects insights charts now use Highcharts, which is an industry standard library for charts, improving our accessibility of projects insights.
- You can now use the [`UpdateProjectV2Field`](https://docs.github.com/en/graphql/reference/mutations#updateprojectv2field) GraphQL API mutation to directly update all single select field options in one API.

## :writing_hand: Tell us what you think!

Share your feedback by posting a comment below 👇 

See how to use GitHub for project planning with [GitHub Issues](https://github.com/features/issues), check out what’s on the [roadmap](https://github.com/orgs/github/projects/4247/views/1), and learn more in the [documentation](https://docs.github.com/en/issues).

**Comment by BenjaminOmar** [comment]
First of all, i would like to say that our team LOVES the new GitHub project structure. There is one feature that would elevate it. As of my understanding, there is no way of doing advanced filtering (like shown above) inside of the project. This would be a great additidon, so we could have even greater KanBan board structures. The field i am talkin about is underlined here:
![Screenshot 2024-12-13 at 09 14 00](https://github.com/user-attachments/assets/6bae243d-e9d3-44e4-a80c-7b5274f454fa)

Would also be great to be able to separatly filter on the sliced properties as stated here:
![Screenshot 2024-12-13 at 09 17 52](https://github.com/user-attachments/assets/ef8778cd-653a-496e-8d45-056348ca0a21)

Other than that, it works great!

**Comment by mr-cheffy** [comment]
HI! This is great, but I dont know why, the "close as duplicate" doesnt show in normal issue pages, only on project pages. ANd when I close as duplicate, it just says I've closed it normally...

Example: https://github.com/zen-browser/desktop/issues/2343#

**Comment by fernandocristan** [comment]
Hi, i had subscribed my org to this feature and after that, my TMetric integration button on issue disappeared on issues pages.
This is a big problem for me, how can I unsubscribe my org for this feature?

**Comment by jordanbrown0** [comment]
"Close as duplicate" is good, but it would be even better if the e-mail notification included the issue number, description, and link for the issue that you're closing this one as a duplicate of.
